### ISSUE TITLE
change log popup window is not accessible

### ISSUE THREAD

**Issue by ghouston** [warpdotdev/Warp#2603]
### Describe the bug

After updating Warp to the latest version, the "Change Log" dialog appears on screen.  This dialog does not have essential keyboard navigation which is required for accessibility reasons.  It should allow the user to navigate using the Tab key between high level elements.  Also the arrow keys should allow navigation in the list.  The "esc" key should close the dialog.

Using keyboard navigation I was unable to close the "Change Log" dialog.   Instead it looks like all the keystrokes are going to the terminal tab below the dialog.

This is an accessibility issue!

<img width="524" alt="Screen_Shot_2023-02-03_at_8_10_00_AM" src="https://user-images.githubusercontent.com/21588/216627127-ea1a9836-4d31-4170-ba76-022fe09b7591.png">

### To Reproduce

1. Update warp
2. The Change Log Dialog appears
3. Try to navigate the dialog using keyboard only, no mouse... you cant!!!
4. Try to close the dialog using keyboard only, no mouse... you cant!!!

Notice that your keystrokes appear to be controlling the current terminal not the dialog!

### Operating System

MacOS

### Warp Version

v0.2023.01.31.08.08.stable_02

### Does this block you from using Warp daily?

No

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for letting us know @ghouston It's certainly something we can bring up with the team as an A11y issue. We'll post any updates here.
